install.sh

```diff
@@ -39,8 +39,7 @@ for exe in $bindir/*; do
 	base=$(basename "$exe")
 
 	if [ -L "$exe" ]; then
-		L=$(ls -dl "$exe")
-		link=$(printf ${L#*-> })
+		link=$(readlink "$exe")
 		"$INSTALL" -Dlm 755 "$link$exec_suffix" "$installdir/$base$exec_suffix"
```

Comment:
Hmm...I originally had `readlink` in this spot in this script, but someone (I can't remember who) made me aware that it is not a part of POSIX, which is why there is this weird combination of `ls` and `printf`, which is portable across POSIX.